server/models/user.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose")
+const { Schema } = mongoose
+const Exercise = require("./exercise")
+
+const userSchema = new Schema(
+  {
+    username: { type: String, required: true },
+    firstName: String,
+    lastName: String,
+    weight: Number,
+    age: Number,
+  },
+  {
+    toJSON: { virtuals: true },
+  }
+)
+// At the top with your dependencies:
+
```

Comment:
Is there a reason why the virtual was commented out? This would probably be a nice virtual field to have so that we can quickly see all the exercises that belong to a particular user. Similarly to how we were able to show all the breads that belonged to a particular baker, we can also show all the exercises that belong to a particular user